parameters/world_modeling/world_model_siza.yaml

```diff
@@ -10,14 +10,15 @@ plugins:
 #  lib: libed_robot_plugin.so
 #  parameters:
 #      robot_name: hero
-#      urdf_rosparam: /robot_description
+#      urdf_rosparam: /hero/robot_description
```

Comment:
Why disabled?

Reply:
Again, focused on free-mode, so this launch file is still WIP.